netsim/roles/host.py

```diff
@@ -0,0 +1,87 @@
+'''
+Create detailed node-level data structures from topology
+
+* Discover desired imagex (boxes)
+* Add default module list to nodes without specific modules
+* Set loopback and management interface data
+'''
+import typing
+
+from box import Box, BoxList
+import netaddr
+
+from ..utils import log
+from .. import data
+from ..augment import devices
+
+from . import select_nodes_by_role
+
+'''
+Create the default static route list from pool prefixes
+'''
+def default_static_route_list(topology: Box) -> list:
+  sr_list = []
+
+  for ap_name,ap_data in topology.addressing.items():
+    if ap_name in ['mgmt','router_id']:
```

Comment:
Perhaps this list should be defined in a YAML file, such that future plugins may modify it if needed?

For example, an ```mlag_p2p``` pool might not be intended to have static routes...